spec/plaindate.html

```diff
@@ -864,9 +864,9 @@ <h1>BalanceISODate ( _year_, _month_, _day_ )</h1>
       <h1>PadISOYear ( _y_ )</h1>
       <emu-alg>
         1. Assert: _y_ is an integer.
-        1. If _y_ &gt; 999 and _y_ ≤ 9999, then
-          1. Return _y_ formatted as a four-digit decimal number.
-        1. If _y_ ≥ 0, let _yearSign_ be *"+"*; otherwise, let _yearSign_ be *"-"*.
+        1. If _y_ ≥ 0 and _y_ ≤ 9999, then
+          1. Return _y_ formatted as a four-digit decimal number, padded to the left with zeroes as necessary.
+        1. If _y_ &gt; 0, let _yearSign_ be *"+"*; otherwise, let _yearSign_ be *"-"*.
```

Comment:
This line should remain unchanged from what it was previously. Otherwise, with this change, year 0 would output `-000000`.

Reply:
This line won't be reached if `y == 0`. Previous step will return `0000` (which is the desired output).

Therefore, `> 0`, `≥ 0` and `> 9999` would be equivalent here. Is `≥ 0` preferable?

Reply:
Oh, you're certainly right. Never mind.

If you think it's likely that more people might make the same mistake as I did, "≥" might be marginally better in that case. But either way is good.

Reply:
I think, in a context of spec (not patch) `≥` would be more likely to _make_ more people assume that 0 is possible at this step. So my preference is to keep the change.

P.S. Overall, reversed condition might be more straightforward and failproof:
```diff
- If offsetNanoseconds ≥ 0, let sign be "+"; otherwise, let sign be "-".
+ If offsetNanoseconds < 0, let sign be "-"; otherwise, let sign be "+".
```